api/openapi.gen.json

```diff
@@ -346,6 +346,9 @@
             },
             "description": "Returned on success."
           },
+          "204": {
```

Comment:
If we are returning 204, then I believe we should remove 200?

Reply:
oh yeah, I've missed we kept it in, nice catch.
@avitova could you remove the 200 response from `path.yml` pls?